spec/webvalve/service_url_converter_spec.rb

```diff
@@ -10,6 +10,24 @@
       expect(subject.template).to be_an(Addressable::Template)
     end
 
+    context "with a regexp" do
+      let(:url) { %r{\Ahttp://foo\.com} }
+
+      it "returns the same object" do
+        expect(subject.template).to be_a(Regexp)
+        expect(subject.template).to equal(url)
+      end
+    end
+
+    context "with an Addressable::Template" do
+      let(:url) { Addressable::Template.new("http://foo.com{/path*}") }
+
+      it "returns a template" do
```

Comment:
this should probably say "returns the same object" as well, right?

Reply:
yes
```suggestion
      it "returns the same object" do
```